src/components/app/MenuButtons/MetaInfo.css

```diff
@@ -2,47 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.menuButtonsMetaInfoButtonBox,
-.menuButtonsMetaInfoButtonLabel,
-.menuButtonsMetaInfoButtonButton {
-  height: 100%;
-  font-size: 100%;
-}
-
-.menuButtonsMetaInfoButtonBox {
+.menuButtonsMetaInfoButton {
   display: flex;
 
   /* Let this box shrink down to the size of the three dots and a bit of text. */
   min-width: 75px;
+  height: 100%;
+  flex-flow: column nowrap;
 
   /* Shrink down twice as fast as the range selection labels. */
   flex-shrink: 2;
   padding: 0;
+  border-left: 1px solid var(--grey-30);
+  font-size: 100%;
 }
```

Comment:
It looks like that there's some more clean-up that we can do here, as some of the properties seem either already defined by `buttonWithPanel` (a class that's also applied by default in this component) or useless:
* `display: flex` and `flex-flow` are already defined
* `height`, `padding` and `font-size` seem ussless.

Can you please remove them?